libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/FeatureList/FeatureList.tsx

```diff
@@ -65,39 +77,76 @@ export class FeatureList extends React.Component<
   IFeatureListState
 > {
   private layerHostId: string;
+  private _selection: Selection;
+  private _theme: ITheme;
```

Comment:
Any reason why were storing this on the object? I'm not a React expert, but would this work if we change the theme halfway through? Or would it still keep the initial theme?

Ideally we should use global context for things like theme, and in the meanwhile `getTheme()` is just fine. @xuke444 may have thoughts.

Reply:
good point!  I put it inside the render function instead.